src/utils/file.js

```diff
@@ -34,3 +37,17 @@ export function getTypedFolderId(id: string): string {
 export function isBoxNote(file: BoxItem): boolean {
     return file.extension === FILE_EXTENSION_BOX_NOTE;
 }
+
+/**
+ * Returns the extension from the file name
+ * @param {string} filename a Box file
+ * @return {string} typed id for file
+ */
+export function getFileExtension(filename: any): string {
```

Comment:
We've been trying to avoid the generic types, like `any`, `Object`, `Function`. You can add `// @flow strict` to the top of your files to assist.